### PR TITLE
[SPARK-339] Add support for Kerberos args to Go CLI module

### PR DESCRIPTION
**Pull request by nickbp**
Tested by launching a job with phony Kerberos args against the Python CLI and then the Go CLI with the same args, then checking the resulting launched job args for differences. From that testing, assuming Python submit works, Go submit should work too..

### CODE REVIEW

**Comment by nickbp** [comment]
build failure: `ERROR: Unable to find project for artifact copy: mesos-spark-integration-tests`

File: cli/dcos-spark/submit_builder.go [hunk 1]
```diff
@@ -204,7 +228,7 @@ ARGLOOP:
 		}
 	}
 	if cli.Verbose {
-		fmt.Printf("Translated arguments: '%s'\n", argsEquals)
+		log.Printf("Translated arguments: '%s'\n", argsEquals)
```
Comment (nickbp) [inline cli/dcos-spark/submit_builder.go]: FYI this change is to move a stdout print to stderr.

**Comment by nickbp** [comment]
retest this please

**Comment by elezar** [comment]
@nickbp Since the JIRA ticket has been closed and this PR is pretty stale, I assume that we can close this? Please do so, and create a follow up ticket if there are any changes that should still be incorporated in our Spark package.

**Comment by nickbp** [comment]
I think this was done in a separate PR. In any case it's very old. Closing.

**Comment by nickbp** [comment]
Yeah it looks like related work was done via #257

**Pull request closed without merging**